### ISSUE TITLE
Does LGSVL Simulator support Nvidia RTX 3090 with Apollo ADS?

### ISSUE THREAD

**Issue by GabrielZZZ** [lgsvl/simulator#1284]
Hello. It is not actually an issue report here. I just want to ask whether this powerful simulator supports running with Apollo ADS on Nvidia RTX 3090 since I remember there was a problem with the perception module of Apollo on the 2080Ti GPU (the car moves slowly & sometimes routing does not move the car). Has that problem been fixed?

Thanks.

**Comment by EricBoiseLGSVL** [comment]
Yes, when using Apollo 6 RTX GPUs are supported.  Apollo 5 does not support RTX GPUs.  FYI Apollo 6 has a bug in perception so we have module testing as a workaround until they fix it.
